package.json

```diff
@@ -53,7 +53,7 @@
     "protractor": "~5.1.2",
     "ts-node": "^3.0.4",
     "tslint": "^5.3.2",
-    "typescript": "~2.3.3",
+    "typescript": "^2.5.2",
```

Comment:
This is only if you set Angular to ^5.1, otherwise set `~2.4.2`.

Reply:
I'm pushing a change in a bit to fix it to only minor versions for angular@5.